php-checks/src/main/java/org/sonar/php/checks/formatting/FunctionSpacingCheck.java

```diff
@@ -82,22 +82,22 @@ private void checkSpaceForComma(FormattingStandardCheck formattingCheck, AstNode
       Token nextToken = comma.getNextSibling().getToken();
       Token previousToken = comma.getPreviousSibling().getLastToken();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "previousToken" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "previousToken". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)